src/modules/pageTable.js

```diff
@@ -27,10 +34,26 @@ class PageTableRow extends React.PureComponent {
                         accessor = column.accessor(row);
                     }
 
+                    const style = {};
+                    const size = (sizes[rowIndex] || [])[index];
+
+                    if (size) {
+                        style.height = size.h;
+                        style.width = size.w;
+                    }
+
+                    if (idPrefix === 'column') {
+                        if (splitter && _.last(columns) === column) {
+                            style.borderRight = '1px solid #edf1f5';
+                        }
+                    }
+
                     return (
                         <Table.Cell
+                            id={`tableCell-${idPrefix}-${rowIndex}-${index}`}
```

Comment:
ID here should probably be `table-cell...` rather than `tableCell` ... we seem to favor a combination of "snake case" and "kebab case" for our IDs and classes rather than "camel case".  But I'll leave the final decision on that to yourself and Mssr. @morethanfire .

Reply:
Let's do `table--table_cell-${idPrefix}_${rowIndex}_${index}`.

Reply:
Done.